modules/prediction/proto/offline_features.proto

```diff
@@ -21,6 +21,7 @@ message DataForLearning {
   optional double timestamp = 2;
   // The features for learning algorithms:
   repeated double features_for_learning = 3;
+  repeated string string_features_for_learning = 7;
```

Comment:
IMO the `string_` prefix is not needed.

Reply:
This is to differentiate from the existing "features_for_learning".